akka-bbb-apps/src/main/scala/org/bigbluebutton/endpoint/redis/LearningDashboardActor.scala

```diff
@@ -30,19 +30,20 @@ case class Meeting(
 )
 
 case class User(
-  userKey:            String,
-  extId:              String,
-  intIds:             Map[String,UserId] = Map(),
-  name:               String,
-  isModerator:        Boolean,
-  isDialIn:           Boolean = false,
-  currentIntId:       String = null,
-  answers:            Map[String,Vector[String]] = Map(),
-  talk:               Talk = Talk(),
-  emojis:             Vector[Emoji] = Vector(),
-  reactions:          Vector[Emoji] = Vector(),
-  webcams:            Vector[Webcam] = Vector(),
-  totalOfMessages:    Long = 0,
+                 userKey:            String,
+                 extId:              String,
+                 intIds:             Map[String,UserId] = Map(),
+                 name:               String,
+                 isModerator:        Boolean,
+                 isDialIn:           Boolean = false,
+                 currentIntId:       String = null,
+                 answers:            Map[String,Vector[String]] = Map(),
+                 pluginAnalytics:            Vector[PluginAnalytics] = Vector(),
```

Comment:
It could be a `Map[String,Vector[PluginAnalytics]] = Map()`.
This way will be easier to group all info of the same plugin inside the same key!